src/expressionEvaluator/expressionEvaluator.ts

```diff
@@ -0,0 +1,30 @@
+import { IExpressionEvaluator } from './iExpressionEvaluator'
+import { ExpressionNodeEvaluator } from './expressionNodeEvaluator'
+import { ExpressionNodeType } from './expressionNodeType'
+import { ExpressionFunction } from './expressionFunction'
+import { ExpressionContext } from './expressionContext'
+import { JSExpr } from './jsExpr'
+import jsep from './jsep'
+
+export class ExpressionEvaluator implements IExpressionEvaluator {
```

Comment:
is this an abstract class?

Reply:
uhm, I still don't know.. for simple expressions it can be used as it is